.gitignore

```diff
@@ -99,3 +99,8 @@ diagrams/*.atxt
 diagrams/plantuml.jar
 
 netdata.cppcheck
+
+/web/dashboard_info.js
+/conf.d/health.d/cpu.conf
```

Comment:
This and the following two lines I do not like. Reason:
- Adding new os-dependend alarms is too complex:
   - Adding the alarm file
   - Adding to Makefile.am (Not obvious because in the root directory)
   - conf.d/Makefile.am (not obvious)
   - Adding to .gitignore (Totally not obvious)

I'll make a proposal how to do this more easy at the end of the review.